src/Extension/CoreExtension.php

```diff
@@ -948,6 +949,20 @@ public static function sortFilter(Environment $env, $array, $arrow = null)
         return $array;
     }
 
+    /**
+     * Shuffle an array.
+     * The function does not preserve keys.
+     *
+     * @internal
+     */
+    public static function arrayShuffle(iterable $array): array
```

Comment:
I think there is no function currently that enforce argument types like this... 

Is this something we want to have in the future ? Right now ? 

I feel that could lead to some unexpected behaviour, as this one would raises a PHP `TypeError` if given a string for example, whereas similar functions/filters would raise a `Twig\Error\RuntimeError`.

WDYT ?

Reply:
Most Twig filters try hard to work on everything you pass them. So, here, we might support strings for instance.

Reply:
I understand both your comments.
I will make sure that this type of error does not occur.
I will modify the filter and add support for the "string" type.